src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
+$gap-size: 48px;
+$card-width: 200px;
+$card-height: 300px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $gap-size;
```

Comment:
your cards are not centered relative to the container, but stick to the left edge when the screen size is reduced
maybe justify-content: center and align-content: center can help